qemu/tests/block_hotplug.py

```diff
@@ -67,12 +68,15 @@ def get_new_disk(disk1, disk2):
     get_disk_cmd = params.get("get_disk_cmd")
     context_msg = "Running sub test '%s' %s"
     device_list = []
+    disk_index = params.objects("disk_index")
```

Comment:
We can get the new disks from var `new_disks`, so it is not necessary to introduce this param.

Reply:
I tried to use new_disks to replace the disk_index, when test for with_plug cases, it can work, all with_plug cases can be passed. But when test for without_plug cases, it cannot work, because no new disk hotpluged when test without_plug cases,  the variant new_disks is none. it will cause disk format error due to disk format need use a disk index number.